openshift/e2e-common.sh

```diff
@@ -234,6 +234,15 @@ EOF
     oc patch knativeserving knative-serving \
         -n "${SERVING_NAMESPACE}" \
         --type merge --patch '{"spec": {"config": {"network": {"internal-encryption": "true"}}}}'
+    oc patch knativeserving knative-serving \
+        -n "${SERVING_NAMESPACE}" \
+        --type merge --patch '{"spec": {"config": {"kourier": {"cluster-cert-secret": "server-certs"}}}}'
+    # Deploy certificates for testing TLS with cluster-local gateway
+    timeout 600 '[[ $(oc get ns $SERVING_INGRESS_NAMESPACE -oname | wc -l) == 0 ]]' || return 1
+    yq read --doc 1 ./test/config/tls/cert-secret.yaml | sed "s/knative-serving/${SERVING_INGRESS_NAMESPACE}/" | oc apply -f -
```

Comment:
I don't want to depend on `yq` but if you use `yq read` then we should use replace the value with `yq`.
```
yq read --doc 1 ./test/config/tls/cert-secret.yaml | yq write - metadata.namespace ${SERVING_INGRESS_NAMESPACE} | oc apply -f -
```

Reply:
Is this based on the yq version 3.x?

Reply:
Using `yq` should be alright. We have it available in CI (see [build image](https://github.com/openshift/knative-serving/blob/release-v1.4/openshift/ci-operator/build-image/Dockerfile#L10)) and we use it in serverless-operator repo so everybody should have this installed on localhost by now.
Yes, this is based on version 3.x.

Reply:
Another option would be to use just sed:
Print first part of the file:
```
sed -e '/^---$/Q' ./test/config/tls/cert-secret.yaml -e "s/knative-serving/${SERVING_INGRESS_NAMESPACE}/"
```
Print second part:
```
sed '0,/^---$/d' ./test/config/tls/cert-secret.yaml
```
But is is less readable

Reply:
Pushed Kenjiro's suggestion.